tests/tests_geomstats/test_binomial.py

```diff
@@ -40,3 +39,41 @@ def test_point_to_pdf(self, n_draws, point, n_samples):
             pmf.append(gs.array([binom.pmf(x, n_draws, point[i]) for x in samples]))
         expected = gs.squeeze(gs.stack(pmf, axis=0))
         self.assertAllClose(result, expected)
+
+
+class TestBinomialMetric(RiemannianMetricTestCase, metaclass=Parametrizer):
+    skip_test_log_after_exp = True
+    # skip_test_exp_after_log = True
+    skip_test_parallel_transport_ivp_is_isometry = True
+    skip_test_parallel_transport_bvp_is_isometry = True
+    # skip_test_geodesic_ivp_belongs = True
+    # skip_test_geodesic_bvp_belongs = True
+    skip_test_exp_geodesic_ivp = True
+    skip_test_exp_ladder_parallel_transport = True
+    # skip_test_triangle_inequality_of_dist = True
+    skip_test_riemann_tensor_shape = NOT_AUTOGRAD
+    skip_test_ricci_tensor_shape = NOT_AUTOGRAD
+    skip_test_scalar_curvature_shape = NOT_AUTOGRAD
+    skip_test_covariant_riemann_tensor_is_skew_symmetric_1 = NOT_AUTOGRAD
+    skip_test_covariant_riemann_tensor_is_skew_symmetric_2 = NOT_AUTOGRAD
+    skip_test_covariant_riemann_tensor_bianchi_identity = NOT_AUTOGRAD
+    skip_test_covariant_riemann_tensor_is_interchange_symmetric = NOT_AUTOGRAD
+    skip_test_sectional_curvature_shape = NOT_AUTOGRAD
+    testing_data = BinomialMetricTestData()
+    Space = testing_data.Space
+
+    def test_squared_dist(self, n_draws, point_a, point_b, expected):
+        self.assertAllClose(
+            self.Metric(n_draws).squared_dist(point_a, point_b), expected
+        )
+
+    def test_metric_matrix(self, n_draws, point, expected):
+        self.assertAllClose(self.Metric(n_draws).metric_matrix(point), expected)
+
+    def test_geodesic_symmetry(self, space_args):
+        space = self.Space(*space_args)
+        point_a, point_b = space.random_point(2)
+        path_ab = space.metric.geodesic(initial_point=point_a, end_point=point_b)
+        path_ba = space.metric.geodesic(initial_point=point_b, end_point=point_a)
+        t = gs.linspace(0.0, 1.0, 10)
+        self.assertAllClose(path_ab(t), path_ba(1 - t))
```

Comment:
Could you add a test to verfiy that `dist` gives the same result as `norm` of `log`? I don't think this test is done automatically since the distance is usually implemented using norm of log, but here it is implemented using a direct formula.

Reply:
Yes I will look into it.

Reply:
@alebrigant There is an automatic `test_dist_is_norm_of_log` where we basically compute and compare `sqrt(squared_dist(point_a, point_b))` and `norm(log(point_b, point_a))`. 

In `RiemannianMetric`, `squared_dist` is computed exactly like you mentionned using `squared_norm` of `log`, however, we override this in `BinomialMetric` using a direct solution so the test is actually done as we wish (using 2 different methods for `dist` and `norm(log)`).

Reply:
Perfect !